src/climatevision/generator/refdata.py

```diff
@@ -121,6 +121,25 @@ def load(
     def rows(self) -> Iterable[tuple[KeyT, list[str]]]:
         return self._rows.items()
 
+    def columns(self, keyword_data: str) -> tuple[str, dict[str, str]]:
+        """Return column of a given keyword."""
+        column_dict = {"": ""}
+        column_dict.update()
+        idx_header = self.header.get(keyword_data, 0)
+        for row in self._rows.items():
+            upd = {str(row[0]): str(row[1][idx_header])}
+            column_dict.update(upd)
```

Comment:
Why not `column_dict[row[0] = row[1][idx_header]`. Also do you want the conversion of all the values to `str` ? Why?